pkg/agent/flowexporter/connections/builder_linux.go

```diff
@@ -0,0 +1,105 @@
+// +build linux
+
+package connections
+
+import (
+	"time"
+
+	"k8s.io/klog"
+
+	"github.com/ti-mo/conntrack"
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+)
+
+const (
+	ctZone        uint16 = 65520
```

Comment:
can we avoid hardcoding the ctZone in multiple different places (already defined in `pkg/agent/openflow/pipeline.go`)?